saleor/graphql/order/types.py

```diff
@@ -265,7 +266,12 @@ def resolve_total_balance(self, info):
 
     @staticmethod
     def resolve_fulfillments(self, info):
-        return self.fulfillments.all().order_by('pk')
+        user = info.context.user
+        if user.is_authenticated and user.is_active and user.is_staff:
```

Comment:
It's enough to check only `user.is_staff` here - if it's true, it implies that the user is authenticated which in turn implies that they are active.

Reply:
```is_staff``` means user is authenticated.

But I think a staff can be not active anymore?

Reply:
Oh, yes, an inactive user can be authenticated at all...

Reply:
But I see there are a lot of code using ```is_active``` in Saleor, maybe let's keep this in case we miss anything?

Reply:
`is_active` flag determines if the user is able to log in at all, so first you have to be "active" in order to determine whether you're staff or not.

Reply:
OK, PR is updated.